public/app/plugins/datasource/loki/components/monaco-query-field/MonacoQueryField.test.tsx

```diff
@@ -31,8 +32,11 @@ function renderComponent({
 describe('MonacoQueryField', () => {
   test('Renders with no errors', async () => {
     renderComponent();
-
+    const user = userEvent.setup();
     const monacoEditor = await screen.findByTestId(selectors.components.ReactMonacoEditor.editorLazy);
     expect(monacoEditor).toBeInTheDocument();
+
+    const editor = await screen.findByLabelText(/editor content/i);
+    await user.type(editor, '1');
```

Comment:
This is just an example failing test with the query editor component, that fails due to the `triggerSuggest` command missing 🤷